### ISSUE TITLE
Loss + Probability Excellence Curves do not display correctly when gamma is altered

### ISSUE THREAD

**Issue by pdunstan88776** [theonaunheim/pyfair#60]
```
from pyfair import FairModel, FairSimpleReport, FairMetaModel
from random import random

gamma1 = 4

model1 = FairModel(name="Model 1 - 4 shape", n_simulations=100_000, random_seed=int(random()*100))

model1.input_data('Contact Frequency', low=0, mode=0.33, high=2)#, gamma=gamma1)
model1.input_data('Probability of Action', low=0, mode=.5, high=1, gamma=gamma1)

model1.input_data('Threat Capability', low=.5, mode=.75, high=.9, gamma=gamma1)
model1.input_data('Control Strength', low=.1, mode=.4, high=.6, gamma=gamma1)

model1.input_data('Primary Loss', low=1_736, mode=58_060, high=2_218_370, gamma=gamma1)

model1.input_data('Secondary Loss Event Frequency', low=0, mode=.2, high=.5, gamma=gamma1)
model1.input_data('Secondary Loss Event Magnitude', low=306_000, mode=1_230_000, high=7_428_250, gamma=gamma1)


model1.calculate_all()


gamma2 = 1

model2 = FairModel(name="Model 2 - 1 shape", n_simulations=100_000, random_seed=int(random()*100))
model2.input_data('Contact Frequency', low=0, mode=0.33, high=2, gamma=gamma2)
model2.input_data('Probability of Action', low=0, mode=.5, high=1, gamma=gamma2)
model2.input_data('Threat Capability', low=.5, mode=.75, high=.9, gamma=gamma2)
model2.input_data('Control Strength', low=.1, mode=.4, high=.6, gamma=gamma2)
model2.input_data('Primary Loss', low=1_736, mode=58_060, high=2_218_370, gamma=gamma2)
model2.input_data('Secondary Loss Event Frequency', low=0, mode=.2, high=.5, gamma=gamma2)
model2.input_data('Secondary Loss Event Magnitude', low=306_000, mode=1_230_000, high=7_428_250, gamma=gamma2)

model2.calculate_all()


fsr = FairSimpleReport([model1, model2])
fsr.to_html('output.html')
```

<img width="1361" alt="Screenshot" src="https://github.com/Hive-Systems/pyfair/assets/153143599/e334d7d4-2f0f-4c55-a869-36a7ccca867c">

**Comment by pdunstan88776** [comment]
<img width="1361" alt="Screenshot" src="https://github.com/Hive-Systems/pyfair/assets/153143599/c292756e-d88c-4007-8f76-e4a8f4330f4a">

**Comment by coreyneskey** [comment]
The behavior you've observed, where gamma=1 (Model 2) produces significantly different and more extreme loss exceedance curves compared to gamma=4 (Model 1), is not a bug but rather the correct and expected mathematical behavior of the BetaPERT distribution.

In the BetaPERT distribution, the gamma parameter (often called the "shape" or "peakedness" parameter) significantly influences the distribution's mean and standard deviation.
Mean: The mean is calculated as (low + gamma * mode + high) / (gamma + 2). A smaller gamma gives less weight to the mode.
Standard Deviation: The standard deviation is calculated as (high - low) / (gamma + 2). This is a crucial point:
For gamma = 4 (Model 1), stdev = (high - low) / 6.
For gamma = 1 (Model 2), stdev = (high - low) / 3.

A smaller gamma leads to a larger standard deviation. A gamma of 1 results in a standard deviation that is twice as large as that with gamma of 4 (assuming low and high are the same).

A larger standard deviation means the distribution is more spread out and has "fatter tails." This implies a higher probability of encountering values further from the mean, including more extreme high values.

Model 1 (gamma=4): The standard gamma value. The distribution is more concentrated, leading to less extreme values in the tail (as seen in your blue curves).

Model 2 (gamma=1): With a smaller gamma, the distribution is wider. This results in a higher likelihood of sampling more extreme loss values, which is why:

1. The "Exceedance Probability Curve" (left plot, orange line) for Model 2 shoots up much more dramatically at the highest percentiles.
2. The "Loss Exceedance Curve" (right plot, orange line) for Model 2 is shifted to the right, indicating a higher probability of exceeding larger loss amounts.

In short, I think Pyfair is correctly implementing the BetaPERT distribution. The gamma parameter provides a way to adjust the shape and spread of the distribution. Your experiment accurately demonstrates that a lower gamma (like 1) will produce a distribution with higher variance and fatter tails, leading to more extreme potential outcomes compared to the standard gamma of 4.

This is an important characteristic of the PERT distribution and highlights the sensitivity of risk models to such distributional parameters. No code fix is needed as the behavior is mathematically sound and intended.
